crossdock/main.go

```diff
@@ -44,7 +44,7 @@ type clientHandler struct {
 }
 
 func main() {
-	agentHostPort = getEnv(envAgentHostPort, "jaeger-agent:5778")
```

Comment:
I am unclear how this even works before this change. There is no `jaeger-agent` anymore in `crossdock/jaeger-docker-compose.yml`. Is this code even being executed? Can we add some logging so that we can see it in the runs?

Reply:
I have added logs , but nothing new pops up when running the build and test script for this .

Reply:
`Dockerfile:10
--------------------
   8 |     ARG TARGETARCH
   9 |     ARG USER_UID=10001
  10 | >>> COPY agent-linux-$TARGETARCH /go/bin/agent-linux
  11 |     EXPOSE 5775/udp 6831/udp 6832/udp 5778/tcp
  12 |     ENTRYPOINT ["/go/bin/agent-linux"]
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref gy29m51xhvo6f6bjl06qu1w8m::sc4007cgbnhy890gx6i98pld0: "/agent-linux-amd64": not found`

Reply:
`ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref gy29m51xhvo6f6bjl06qu1w8m::sc4007cgbnhy890gx6i98pld0: "/agent-linux-amd64": not found`
This seems to be error from the DockerFile when running build and test script .

Reply:
what do you mean - are you running something manually?

Reply:
Its from crossdock ci test

Reply:
I checked the docker compose logs , I got the log for initialization of agent and collector service but no log for fetching sampling rate by the agent .